backend/src/main/java/de/bund/digitalservice/ris/norms/adapter/input/restapi/schema/ProprietaryResponseSchema.java

```diff
@@ -18,4 +21,28 @@ public class ProprietaryResponseSchema {
   public static class Fna {
     private String value;
   }
+
```

Comment:
@VictorDelCampo FYI: I have not changed the general setup of the response schema for my task. This stays an open topic.

At least we now have some more metadata in here to lead whatever abstraction we may use in the future.